content/en/database_monitoring/guide/managed_authentication.md

```diff
@@ -88,6 +125,8 @@ instances:
  aws:
    instance_endpoint: example-endpoint.us-east-2.rds.amazonaws.com
    region: us-east-2
+   managed_authentication:
+    enabled: true
```

Comment:
I have a pending PR to update this but has to hold on until 7.51 is released. remove for now?

Reply:
This can be added to postgres & shouldn't matter right now right? The current agent will just look at the `region`. We can update the azure configuration after the agent is released